src/main/content/guides.html

```diff
@@ -65,8 +50,10 @@ <h3 class="guide_title">{{guide.title}}</h3>
                     <img class="duration_clock_icon" src="/img/guide_duration_clock_icon_small.svg" alt="Duration">
                     <span class="guide_duration">{{guide.duration}}</span>
                     {% if guide.layout == 'interactive-guide' %}
-                    <img class="interactive_bolt_icon" src="/img/guide_lightning_bolt.svg" alt="Interactive">
-                    <span class="guide_interactive">Interactive</span>
+                    <div class="guide_interactive_container">
+                            <img class="interactive_bolt_icon" src="/img/guide_lightning_bolt.svg" alt="Interactive">
```

Comment:
Do we still need class `interactive_bolt_icon`?  I noticed it was deleted in `guide-card.css`

Reply:
Yes, the only thing removed was a margin.